### ISSUE TITLE
config load: "Loopback0: {}" throws exception in interfaces.j2 and borks eth0

### ISSUE THREAD

**Issue by jab-r** [sonic-net/sonic-utilities#854]
Problem: show interfaces status was blank for Mellanox SN2700 running 201811

In config_db.json

```
    "LOOPBACK_INTERFACE": {
          "Loopback0":{},
          "Loopback0|10.1.0.1/32": {}
      }, 
```
causes interfaces.j2 to throw an exception. This causes the effect of borking eth0 and disabling ssh access into the switch. Removing the blank Loopback0 solves the issue.
    `$ sudo sonic-cfggen --print-data  -j /etc/sonic/config_db.test.json` DOES NOT show an error.

(I don't know where the blank Loopback0 came from, I didn't edit this section.)

I think this is related to issue #805 in that validation would help.

**Comment by prsunny** [comment]
Were you running `master/201911` on the device and then downgraded to `201811` using the same `config_db.json`? If so, it will not work.

**Comment by jab-r** [comment]
No, I was having issues with `master/201911` reading the eeprom (`/var/run/hw-management` was blank). `201811` was a fresh install from `onie`
